src/components/MyReview.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import ApiService from "../api/ApiService";
+import { FaStar } from "react-icons/fa6";
+
+
+function MyReview() {
+    const [reviewList, setReviewList] = useState([]);
+    const FilledStar = () => <FaStar className="fill-my-color text-3xl inline-block mb-2" />;
```

Comment:
별점 icon으로 하면 clipPath가 안되서 svg로 바꿨습니다,,,,
```suggestion
<svg
                  key={idx}
                  width="50"
                  height="50"
                  viewBox="0 0 14 14"
                   >
                  <path
                    d="M9,2l2.163,4.279L16,6.969,12.5,10.3l.826,4.7L9,12.779,4.674,15,5.5,10.3,2,6.969l4.837-.69Z"
                    transform="translate(-2 -2)"
                  />
                </svg>
```